src/openforms/js/components/admin/form_design/FormSteps.js

```diff
@@ -2,6 +2,7 @@ import React, {useState} from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import {FormattedMessage} from 'react-intl';
+import {FormStepContext} from './Context';
```

Comment:
please group/order the imports with the same rules as python - this should go at the end of the imports with the other relative imports

Reply:
Ah I thought that `prettier` would order them so I wasn't paying attention it, will do so in the future :stuck_out_tongue:

Reply:
looks like there's a plugin for that: https://dev.to/diballesteros/how-to-quickly-sort-imports-with-prettier-3po7! I created #2487 for this

Reply:
Oh nice, I'll pick that one up